SPECS/xapi.spec

```diff
@@ -45,6 +45,8 @@ Patch1002: xapi-1.249.3-open-openflow-port.XCP-ng.patch
 # Drop this patch when we don't want to support migration from older SDN controller anymore
 Patch1003: xapi-1.249.3-update-db-tunnel-protocol-from-other_config.XCP-ng.patch
 Patch1004: xapi-23.3.0-filter-link-local-address-ipv6.XCP-ng.patch
+# This patch is available upstream
+Patch1005: xapi-23.3.0-Allow-a-user-to-select-on-which-SR-to-run-quicktest.patch
```

Comment:
Let's mention the lower version of XAPI which has the patch. This will help knowing when to drop it. I also added .backport to the file name, per our patch naming convention (ok, it's [hidden in the forum](https://xcp-ng.org/forum/topic/183/git-repository-structure-and-conventions-for-rpms), though there IS a way to get to it from our official packaging docs)

```suggestion
# Backport from XAPI v23.21.0 and above
Patch1005: xapi-23.3.0-Allow-a-user-to-select-on-which-SR-to-run-quicktest.backport.patch
```

(Applying this also means renaming the file, of course).